mobilebot/mobilebot.c

```diff
@@ -149,11 +151,21 @@ void read_mb_sensors(){
     // Read encoders    
     mb_state.left_encoder_delta = LEFT_ENCODER_POLARITY * rc_encoder_read(LEFT_MOTOR);
     mb_state.right_encoder_delta = RIGHT_ENCODER_POLARITY * rc_encoder_read(RIGHT_MOTOR);
+    mb_state.left_velocity = ((float) mb_state.left_encoder_delta * (WHEEL_DIAMETER * 3.141) / (GEAR_RATIO * ENCODER_RES))/(DT);
+    mb_state.right_velocity = ((float) mb_state.right_encoder_delta * (WHEEL_DIAMETER * 3.141) / (GEAR_RATIO * ENCODER_RES))/(DT);    
```

Comment:
@sansaldo can you use PI here?

Reply:
Good catch, fixed it.

Reply:
closing out this comment as we are getting unexpected behavior with M_PI